lesson14/main.js

```diff
@@ -71,6 +71,12 @@ const init = async () => {
   createNewList(val);
 };
 
+const getNumbers = () => {
+  const inputNumberId = document.getElementById("js-number")
+  const inputNumberValue = inputNumberId.value;
+  return inputNumberValue;
+}
+
 openModal.addEventListener("click", () => {
   modal.classList.remove("hidden");
   mask.classList.remove("hidden");
```

Comment:
```
modal.classList.remove("hidden");
mask.classList.remove("hidden");
```

These processes are also used elsewhere.
Why don't you create them as functions?

Reply:
You're right. I implemented new function for them as a `hiddenModalItems`.
Do you think it needs `Items` to the end of variable name? I added it becouse they are include `mask` too. But I'm still thinking...

And I also implemented `showModalItems` as `loading` and `loaded`.

Reply:
> And I also implemented showModalItems like as loading and loaded.

In this time case but.

```
modal.classList.remove("hidden");
mask.classList.remove("hidden");
```

I thought that this process does not need to be a separate function.
because , This is such a detailed process, and and it is not a process that is done more than once.

Reply:
I think it's not bad.
Personally, I thought is also good better `hiddenModal.`

there is no right answer to this, so I think you should decide what you are comfortable with...

Reply:
> And I also implemented showModalItems like as loading and loaded.
In this time case but.

```
modal.classList.remove("hidden");
mask.classList.remove("hidden");
```

I thought that this process does not need to be a separate function.
because , This is such a detailed process, and and it is not a process that is done more than once.

Reply:
Thanks. So I'm going to remove `Items` becouse it's more confortable for me. 

Only if we use something just one time, we don't have to make function for it, right? :) I got it!
By the way you're talking about not 
```
modal.classList.remove("hidden");
mask.classList.remove("hidden");
```
but
```
modal.classList.add("hidden");
mask.classList.add("hidden");
```
is it correct? 
Becouse the latter is used twice but the former is used only once.

Reply:
yes, you are right.

In the case of lesson 14, as for the `showModal`, 
I feel that there is no need to take the trouble make it a separate function.

Because,

* It is processed only once
* The process is light

However, to be honest, I either one is fine whether it is a function or not. 
I think it's easy to read even if there is a function called `showModal` which is the counterpart of `hiddenModal`. 
So I don't think it's wrong to make it a function.

It's up to you to decide.

Reply:
Thank you for your explanation. 
I had been thinking about whether to change it or not, but this time I decided to keep this one.
Becouse I already defined it and `loding` & `loaded`, so this time I'm going to follow them here.

But next time I will do in your idea because I don't want to do something if it doesn't be needed. Becouse I'm lazy.. :)